README.md

```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| 13 | What is the appraisal cycle in the company?|
```

Comment:
Hi @worklove, sorry for all the confusion, this question has already been mentioned in the list. Question no 14 already ask about appraisal
>How is appraisal process here and how it is evaluated (How frequent performance is evaluated)?

could you please change the question, we would be happy to merge it.